### PR TITLE
Implementation of nextPage getter for refactor + unit test

### PR DESCRIPTION
**Pull request by jarmoza**
This PR addresses #260. This getter makes use of the annotation tool's implicit page order. E.g. Given the current page of the app, what is the next (or following) page?

The `next-page` component has been altered to make use of the new getter name style `get*`.

The unit test sets each page as the current page and makes sure the proper 'next page' is being returned from the getter.

### CODE REVIEW

**Review by rmanaem — CHANGES_REQUESTED** [review]
Everything looks good! I added a small comment and requested a quick fix for this function:

https://github.com/neurobagel/annotation_tool/blob/16953c1be2b7c98c1cd7d562b597d65ac0dbdd0d/components/next-page.vue#L78-L82

I think the function should be renamed to `getNextPageButtonColor` and the return statement should be `return this.pageAccessible(this.getNextPage) ? "success" : "secondary"; `

File: components/next-page.vue [hunk 1]
```diff
@@ -14,10 +14,10 @@
             <b-button
                 class="float-right"
                 data-cy="button-nextpage"
```
[the remark this answers is not in the document]
Comment (jarmoza) [reply components/next-page.vue:16]: In this case no, just because it's referring to the 'Next page' button rather than the getter.

**Comment by jarmoza** [comment]
Thanks @rmanaem! For some reason can't reply to your first comment - but good catch re: the call to next page. In the case of the attribute and function, however, these are referring to the 'Next page' button itself so no need to rename it in line with the store getter.

**Comment by jarmoza** [comment]
Should be good now, @rmanaem. Let me know what you think re: my replies.

**Comment by rmanaem** [comment]
I see, I thought `nextPageButtonColor` function was referenced here before
https://github.com/neurobagel/annotation_tool/blob/9519f55719c03331549edac645cedd9a5966a646/components/next-page.vue#L19

**Comment by jarmoza** [comment]
@rmanaem Good catch. That's a find and replace error – should still be `nextPageButtonColor`. I just fixed and committed.

**Pull request merged**